### ISSUE TITLE
We need a base Drupal 8 installation

### ISSUE THREAD

**Issue by thomscode** [pacifica/pacifica-search-drupal#3]
Drupal 8 can be both installed and managed with [composer](https://getcomposer.org/). [Drupal](https://drupal.org) provides a [composer package](https://github.com/drupal-composer/drupal-project) that can give us a starting point.

**Comment by thomscode** [comment]
We need to use [Drupal's composer template](https://github.com/drupal-composer/drupal-project) to start a Drupal 8 project managed by [composer](https://getcomposer.org/).

To do this we should run the following command:
```bash
composer create-project drupal-composer/drupal-project:8.x-dev pacifica-search --stability dev --no-interaction
```

**Comment by thomscode** [comment]
Resolved by PR #7
